server/test/unit/server/webservices/views/test_consumers.py

```diff
@@ -0,0 +1,1377 @@
+import json
+import unittest
+
+import mock
+from django.http import HttpResponseBadRequest, HttpResponseNotFound
+
+from base import assert_auth_CREATE, assert_auth_DELETE, assert_auth_READ, assert_auth_UPDATE
+from pulp.server.exceptions import (InvalidValue, MissingResource, MissingValue,
+                                    OperationPostponed, UnsupportedValue)
+from pulp.server.webservices.views import consumers
+from pulp.server.webservices.views.consumers import (ConsumersView, ConsumerBindingsView,
+                                                     ConsumerBindingResourceView,
+                                                     ConsumerContentActionView,
+                                                     ConsumerContentApplicabilityView,
+                                                     ConsumerContentApplicRegenerationView,
+                                                     ConsumerHistoryView, ConsumerProfilesView,
+                                                     ConsumerProfileResourceView,
+                                                     ConsumerResourceView,
+                                                     ConsumerResourceContentApplicRegenerationView,
+                                                     UnitInstallScheduleView,
+                                                     UnitInstallScheduleResourceView)
+
+
+class TestConsumersView(unittest.TestCase):
+    """
+    Test consumers view.
+    """
+    @mock.patch('pulp.server.webservices.views.consumers.serialization.binding.serialize')
+    @mock.patch('pulp.server.webservices.views.consumers.factory.consumer_bind_manager')
+    def test_expand_consumers(self, mock_factory, mock_serial):
```

Comment:
I would put this test in its own class. I don't think it really belongs in `ConsumersView`.